README.md

```diff
@@ -2,4 +2,38 @@
 
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-devtools.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-devtools)
 
-An in-browser javascript module that helps developers view and configure OpenMRS frontend
\ No newline at end of file
+An in-browser javascript module that helps developers view and configure OpenMRS frontend
+
+## Running
+
+After cloning or downloading this repo to your computer:
+
+* Install modules by running:
+`npm install`
+
+* Run:
+`npm start <PORT>`
```

Comment:
`npm start -- --port <PORT>`, since [this line](https://github.com/openmrs/openmrs-esm-devtools/blob/master/package.json#L5) no longer has `--port` in it

Reply:
resolved this